packages/block-library/src/columns/edit.native.js

```diff
@@ -69,110 +77,109 @@ const DEFAULT_COLUMNS_NUM = 2;
  */
 const MIN_COLUMNS_NUM = 1;
 
-/**
- * Maximum number of columns in a row
- *
- * @type {number}
- */
-const MAX_COLUMNS_NUM_IN_ROW = 3;
-
-const BREAKPOINTS = {
-	mobile: 480,
-	large: 768,
-};
-
 function ColumnsEditContainer( {
 	attributes,
 	updateAlignment,
 	updateColumns,
 	columnCount,
 	isSelected,
-	onAddNextColumn,
 	onDeleteBlock,
 	innerColumns,
 	updateInnerColumnWidth,
+	parentBlockAlignment,
+	editorSidebarOpened,
 } ) {
 	const [ resizeListener, sizes ] = useResizeObserver();
 	const [ columnsInRow, setColumnsInRow ] = useState( MIN_COLUMNS_NUM );
+	const [ tempWidth, setTempWidth ] = useState( 0 );
+	const screenWidth = Math.floor( Dimensions.get( 'window' ).width );
+	const { globalStyles } = useContext( GlobalStylesContext );
```

Comment:
I'm not sure if there is `globalStyles` value in the context.

Reply:
Prolly you are right, I will wrap the it within `GlobalStylesContext.Consumer`